### DISCUSSION TITLE
Making code executable in place

### DISCUSSION THREAD

**Discussion by joelmiller** [executablebooks/meta#1145] [Q&A]
I'm trying to follow the instructions at https://jupyterbook.org/en/stable/interactive/thebe.html, but in the introduction it says to click the 'live code' button on the page.  I can't find that.  Am I missing something?  It does have a warning that it's experimental - was it perhaps dropped?

**Comment by agoose77** [comment]
@joelmiller thanks for this report! There was a bug in sphinx-thebe (for Sphinx 7) that was recently fixed. The jupyter-book `stable` documentation has not been built since then, so it appears to be broken. However, [the `latest` documentation](https://jupyterbook.org/en/latest/interactive/thebe.html) was built from the fixed version of `sphinx-thebe`, and should behave as described!
